regression-tests.nobackend/tinydns-data-check/expected_result

```diff
@@ -1,4 +1,4 @@
-17a1219ca79020c1c67ad173a291d3c8  ../regression-tests/zones/example.com
+9f88091d3d1b793bf15a2cf5015bf92d  ../regression-tests/zones/example.com
```

Comment:
data.cdb should be regenerated and the checksum below updated. Once that is done, it is possible you won't need `skip.tinydns` at all. `description` in this dir has docs on the regenerating.

Reply:
I'll have a look, but it looks like auth tests now succeed.
In `generate-data.sh`, what is the `tcpclient` tool ?

Reply:
it comes from the ucspi-tcp package from djb (http://cr.yp.to/). The tests indeed pass here when they sometimes should not.